gazebo_ros2_control/src/gazebo_ros2_control_plugin.cpp

```diff
@@ -197,8 +201,10 @@ void GazeboRosControlPlugin::Load(gazebo::physics::ModelPtr parent, sdf::Element
   rclcpp::Duration gazebo_period(impl_->parent_model_->GetWorld()->Physics()->GetMaxStepSize());
```

Comment:
This might not affect much but I realized that [`rclcpp::Duration`](https://docs.ros2.org/foxy/api/rclcpp/classrclcpp_1_1Duration.html) constructor is using **nanoseconds** while [`GetMaxStepSize()`](http://osrf-distributions.s3.amazonaws.com/gazebo/api/11.0.0/classgazebo_1_1physics_1_1PhysicsEngine.html#a0113a9c364f1d7fb9b2072161bec82fa) returns **seconds** in `double` format.

```suggestion
  rclcpp::Duration gazebo_period(impl_->parent_model_->GetWorld()->Physics()->GetMaxStepSize() * 1e9);
```